Source/Mac/Preferences/PreferencesHelper.swift

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2004-2012 Lukhnos Liu (lukhnos at openvanilla dot org)
```

Comment:
Nit: seems to be a truncated header; use the one in [AdvancedPreferencesViewController.swift](https://github.com/openvanilla/openvanilla/pull/114/files#diff-9efe573855b59767a4e5dc2d373faf1bfa45eb5399c40707446f9050c8635f00)?